app/test_main.py

```diff
@@ -1,3 +1,54 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalsYearsTransfer:
+    @pytest.mark.parametrize(
+        "cat_years,dog_years,expected_results",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return [0, 0] if animals ages are 0's too."
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return [0, 0] if animals ages are less then 15"
+            ),
+            pytest.param(
+                17,
+                17,
+                [1, 1],
+                id="should return [1, 1] if animals ages are >= 15 and < 23"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
```

Comment:
The expected result `[2, 2]` for `cat_years` 27 and `dog_years` 27 may not align with the comment description. Ensure that the logic in `get_human_age` matches this expectation, as the comment suggests a specific range for `cat_years` and `dog_years`.